krikri.gemspec

```diff
@@ -27,6 +27,7 @@ Gem::Specification.new do |s|
   s.add_dependency "jsonpath"
   s.add_dependency "devise", "~>3.4.1"
   s.add_dependency "resque", "~>1.0"
+  s.add_dependency "dpla-couchrest", "1.2.1.pre.dpla"
```

Comment:
I pushed up a version of `dpla-couchrest` to [Rubygems](https://rubygems.org/gems/dpla-couchrest), and we have a [repo](https://github.com/dpla/couchrest). The only change that I made was to update the dependencies to avoid the `mime-types` version conflict that someone tried to address with couchrest/couchrest#103.